src/containers/pages/JurisdictionAssignment/helpers/SelectedJurisdictionsCount/index.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { Fragment, useState } from 'react';
+import { Tooltip } from 'reactstrap';
+import { TreeNode } from '../../../../../store/ducks/opensrp/hierarchies/types';
+
+/** Props for SelectedJurisdictionsCount  */
+export interface SelectedJurisdictionsCountProps {
+  id: string;
+  jurisdictions: TreeNode[] /** array of jurisdictions */;
+  parentNode: TreeNode | undefined;
```

Comment:
this is equivalent to `parentNode?` - why not do that?  Or `TreeNode | null`?

Reply:
With the selector implementation it has to be define that way